reflex/components/core/breakpoints.py

```diff
@@ -0,0 +1,74 @@
+"""Breakpoints utility."""
+
+from typing import Optional, Tuple
+
+breakpoints_values = ["30em", "48em", "62em", "80em", "96em"]
+
+
+def set_breakpoints(values: Tuple[str, str, str, str, str]):
+    """Overwrite default breakpoint values.
+
+    Args:
+        values: CSS values in order defining the breakpoints of responsive layouts
+    """
+    breakpoints_values.clear()
+    breakpoints_values.extend(values)
+
+
+class Breakpoints(dict):
+    """A responsive styling helper."""
+
+    def __init__(self, mapping: dict):
+        """Instantiate breakpoints mapping.
+
+        Args:
+            mapping: a dictionary where breakpoints are keys and CSS values are values
+        """
+        super().__init__(mapping)
```

Comment:
is this function necessary? or is it mostly for the docstring?

Reply:
It's used in style.py once. (and in, that file, but that can be replaced)

Reply:
i just meant since it's inherited from the base class `dict`, it should still work even if its not overridden here.

Reply:
you are correct, will fix this then